jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/gradle/GradleProjectPropertiesTest.java

```diff
@@ -366,6 +376,11 @@ public void testCreateContainerBuilder_correctFiles()
             applicationDirectory.resolve("classes/HelloWorld.class"),
             applicationDirectory.resolve("classes/some.class")),
         layers.classesLayerEntries.get(0).getLayerEntries());
+
+    assertModificationTime(Instant.ofEpochSecond(32), layers.snapshotsLayerEntries);
+    assertModificationTime(Instant.ofEpochSecond(32), layers.dependenciesLayerEntries);
+    assertModificationTime(Instant.ofEpochSecond(32), layers.resourcesLayerEntries);
+    assertModificationTime(Instant.ofEpochSecond(32), layers.classesLayerEntries);
```

Comment:
I'd prefer this magic value be a constant — I was a bit surprised by this and had to go hunting to find the source.